api.md

```diff
@@ -171,7 +172,115 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_e
 ```
 
 ---
-## attach_tablespace() [](attach_tablespace)
+### add_data_node() [](add_data_node)
+
+Create a new data node for use when creating distributed hypertables.
+
+To create a distributed hypertable it is necessary to define the data
+nodes where the data is going to be stored, which this utility is used
+for.
+
+The utility will attempt to bootstrap the data node by creating the
```

Comment:
I would elide this part if you go with my suggestion above.

Reply:
Yup.